provision/infinispan/ispn-helm/templates/infinispan.yaml

```diff
@@ -55,8 +55,12 @@ spec:
   expose:
     type: Route
   configMapName: "cluster-config"
+  {{- if .Values.image }}
   image: {{ .Values.image | default .Values.defaultImage }}
-  version: 15.0.4
+  {{- end}}
+  {{- if .Values.version }}
+  version: {{ .Values.version }}
+  {{- end}}
```

Comment:
Only use custom image and version in case these values are explicitly provided via Helm - otherwise use Operator-defaults.